sdk/tools/mkhive/reginf.c

```diff
@@ -124,7 +124,7 @@ append_multi_sz_value(
     IN HKEY KeyHandle,
     IN PCWSTR ValueName,
     IN PCWSTR Strings,
-    IN ULONG StringSize) // In characters
+    IN SIZE_T StringSize) // In characters
```

Comment:
Why?

Reply:
Entire inflib has been changed to accept `SIZE_T` instead of `ULONG` for string sizes, which makes sense considering that the output of `strlen`/`wcslen` or similar functions is likely fed into such variables at some point.